app/models/podcast_import.rb

```diff
@@ -250,7 +250,7 @@ def create_story(entry, series)
       title: clean_string(entry[:title]),
       short_description: clean_string(short_desc(entry)),
       description_html: entry_description(entry),
-      tags: entry[:categories],
+      tags: Array(entry[:categories]).map(&:strip),
```

Comment:
should we also remove nil/empty elements?
`Array(entry[:categories]).map(&:strip).reject(&:blank?)`